index.js

```diff
@@ -233,7 +237,29 @@ module.exports = function (options) {
 
   router.get('/', function (req, res) {
     // If we've gotten this far then token gives proper access and that's all that this route care about
-    return res.sendStatus(200);
+    var isSyndicationRequest = (req.query.q === 'syndicate-to');
+    logger.debug({'query': req.query}, 'Received a query');
+
+    if (isSyndicationRequest) {
+      res.type('application/x-www-form-urlencoded');
+      var data = '';
+      for (var i = options.syndication.length - 1; i >= 0; i--) {
+        var syndicateTo = options.syndication[i];
+        data += 'syndicate-to[]=' + syndicateTo + '&';
+      }
+      res.format({
+        'application/x-www-form-urlencoded': function () {
+          return res.send(encodeURI(data));
```

Comment:
If one uses `QueryString.stringify()` or `encodeURIComponent()` as I noted earlier then there's no need for an `encodeURI()` here.